.eslintrc.js

```
@@ -283,6 +283,12 @@ module.exports = {
         "react/forbid-component-props": forbiddenDomPropsConfig,
       },
     },
+    {
```

No reason to not allow `!` assertions in tests. If it doesn't exist, the test should throw an error anyway.

---

You can add this to the shared config’s `./tests.js` if it’s not already there